receiver/awscontainerinsightreceiver/internal/efa/efaSysfs_test.go

```diff
@@ -10,6 +10,8 @@ import (
 	"testing"
 	"time"
 
+	"golang.org/x/exp/slices"
```

Comment:
This should be part of the second group. It's failing linting.

Reply:
Sorry, addressed now.